internal/rukpak/convert/registryv1_test.go

```diff
@@ -418,6 +420,17 @@ var _ = Describe("RegistryV1 Suite", func() {
 			})
 		})
 
+		Context("Should read the registry+v1 bundle filesystem correctly", func() {
+			It("should include metadata/properties.yaml and csv.metadata.annotations['olm.properties'] in chart metadata", func() {
+				fsys := os.DirFS("testdata/combine-properties-bundle")
+				chrt, err := RegistryV1ToHelmChart(context.Background(), fsys, "", nil)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(chrt).NotTo(BeNil())
+				Expect(chrt.Metadata).NotTo(BeNil())
+				Expect(chrt.Metadata.Annotations).To(HaveKeyWithValue("olm.properties", `[{"type":"from-csv-annotations-key","value":"from-csv-annotations-value"},{"type":"from-file-key","value":"from-file-value"}]`))
+			})
+		})
+
```

Comment:
ACK! We _didn't_ get rid of Ginkgo everywhere‽‽

Reply:
Or we did, but it came back when rukpak was pulled in.

Reply:
https://github.com/operator-framework/operator-controller/issues/1496

Reply:
And I was part of that... oh well...